tests/python/frontend/tflite/test_forward.py

```diff
@@ -1099,11 +1099,22 @@ def _test_reduce_sum(data, keep_dims=None):
     """ One iteration of reduce_sum """
     return _test_reduce(math_ops.reduce_sum, data, keep_dims)
 
+#######################################################################
+# Reduce_any
+# -----------
+
+def _test_reduce_any(data, keep_dims=None):
+    """ One iteration of reduce_any """
+    return _test_reduce(math_ops.reduce_any, data, keep_dims)
 
-def _test_forward_reduce(testop):
+
+def _test_forward_reduce(testop, dtype=None):
     """ Reduce """
     data0 = [np.random.rand(16, 16, 16, 16).astype("float32"), None]
     data1 = [np.random.rand(16, 16, 16, 16).astype("float32"), np.array([1, 2], dtype=np.int32)]
+    if dtype == 'bool':
+        data0[0] = (data0[0] < 0.5).astype(dtype)
+        data1[0] = (data1[0] < 0.5).astype(dtype)
```

Comment:
It seems a little strange. Wait for other review's opinions.

Reply:
To generate boolean numpy boolean data you can use 
`data = np.random.choice(a=[False, True], size=(1, 2, 3)).astype('bool')`